pyveg/src/coordinate_utils.py

```diff
@@ -8,10 +8,29 @@
 import requests
 
 
-def get_region_string(coords, region_size):
+def get_coords(bounds):
     """
-    Given a set of (long,lat) coordinates, and the size
-    of a square region in long,lat space, return
+    Given a bounding box of coordinates,  return
+    the centre point coordinates.
+
+    Parameters
+    ==========
+    bounds: list of lists with coordinates, e.g.  [left, bottom, right, top]]
+
+    Returns
+    =======
+    coords: list of floats, [longitude,latitude]
+    """
+
+    x = (bounds[0] - bounds[2]) / 2 + bounds[0]
+    y = (bounds[3] - bounds[1]) / 2 + bounds[1]
+
+    return [x, y]
+
+
+def get_region_string(bounds):
```

Comment:
Same as the comment below, leaving these functions there for now in the utils, just in case we want to use them later. But we should remove them once the pipeline is fixed.